sql/core/src/main/scala/org/apache/spark/sql/jdbc/PostgresDialect.scala

```diff
@@ -85,6 +85,10 @@ private object PostgresDialect extends JdbcDialect {
     s"SELECT 1 FROM $table LIMIT 1"
   }
 
+  override def getTruncateQuery(table: String): String = {
+    s"TRUNCATE ONLY $table"
```

Comment:
Could you add a function comment about why we use `TRUNCATE ONLY` here.

Reply:
Also, please add the similar explanation to `JdbcDialect.scala`, too.

Reply:
yes will do

Reply:
Done